app/controllers/manager/venues_controller.rb

```diff
@@ -2,6 +2,7 @@ class Manager::VenuesController < ApplicationController
 
   before_action :set_venue, only: [:edit, :show, :update]
   before_action :require_user
+  before_action :require_correct_manager, only: [:edit, :show, :update]
```

Comment:
My reasoning was that the 'new' action wasn't dependent on anything, we could limit it so that no one could start the process. The 'create' action was creating through the current user, didn't think we should limit this but I guess a standard user would still have access. Changing those now, i feel limiting them all is the safest bet.